The Crashed Starwalker

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This PR adds a beach ruin based around a ship I've previously made, called the "Starwalker"

![2023 01 16-16 33 48](https://user-images.githubusercontent.com/79304582/212715120-1234050a-b91c-411c-b792-82d0621cc549.png)

![2023 01 16-16 35 19](https://user-images.githubusercontent.com/79304582/212715457-6b643815-ab0f-4962-9222-1a0d6eeb7535.png)


it contains:
some medical supplies ( oinment slurry / herbal pack / crew monitor / health scanner / charcoal bottle / misc pills )
one Swat suit
one shotgun / one energy cutlass
goliath cloak  / military rig
3 abandoned crates
1 gold crate / one silver crate
lizard wine
one baby carp
a radiant dance machine
a sci protolathe
misc salvage


Lore bit:
After a "most excellent robbery that went like, totally as planned", our protagonists aboard the Starwalker fled the crime scene, with heavy damage to the ship's hull. With one of the Engine blocks almost falling off, The valiant crew decided that the best course of action would be a "Totally rad emergency landing". This, of course, ended in disaster, as the pilot was high on LSD. 
The pilot did however manage to steer them towards a nearby lak- sike, it's just some shallow water. Crashing directly onto the ground, the ship split into multiple fragments, Killing the pilot and crewmate, and Impaling the captain. 
The captain knew that he didn't have long until the bloodloss would get to him, and started moving all his treasure into a nearby cavern. _THERE'S NO WAY_ he would die in that godforsaken ship, nor without his treasures. This is where you now find him, rotting in his "100% real Cow skin" throne _(spacemart Brand Comfy chair)_ . 
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
currently there's a bit of a lack in beach ruins, something that I'd like to help resolve!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Adds a new Beach ruin, the beach_crashed_starwalker
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
Create beach_crashed_starwalker.dmm

spockye committed:
misc changes

spockye committed:
misc files and update

spockye commented:
uhh, something's weird

spockye commented:
I have left your realm, mortals. my PR's exist beyond the limits of your barbaric machines. Look upon my PR's and despair, for you shall never witness their beauty  //edit : Fuck

spockye committed:
more broken glass, and damage

spockye committed:
fixes shutter names

spockye committed:
adds forgotten Datum

spockye committed:
fixes shutters

stacked shutter, with uncapitalized name

spockye committed:
removes docking port

this ship is NOT meant to fly

Erikafox commented on _maps/RandomRuins/BeachRuins/beach_crashed_starwalker.dmm:
```diff
@@ -0,0 +1,5515 @@
+	icon_state = "4-8"
+	},
+/obj/item/gps{
+	gpstag = "NTREC1";
```
iirc this is the kilo's tag too. I'd give em a different one

Erikafox commented on _maps/RandomRuins/BeachRuins/beach_crashed_starwalker.dmm:
```diff
@@ -0,0 +1,5515 @@
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/dirt,
+/obj/item/circuitboard/computer/rdconsole,
+/obj/item/circuitboard/machine/protolathe/department/science{
```
it's just a circuitboard but is the science one appropriate for this ruin? I'd say like... Peacekeeping for ammo (pirate ship) or a cargo one.

Erikafox commented on _maps/RandomRuins/BeachRuins/beach_crashed_starwalker.dmm:
```diff
@@ -0,0 +1,5515 @@
+	resize = 0.5;
+	melee_damage_lower = 0;
+	melee_damage_upper = 0;
+	name = "Baby carp"
```
give this guy a desc too

Erikafox commented on _maps/RandomRuins/BeachRuins/beach_crashed_starwalker.dmm:
```diff
@@ -0,0 +1,5515 @@
+/turf/open/floor/plasteel/stairs{
+	barefootstep = "woodbarefoot";
+	color = "#A47449";
+	footstep = "wood"
```
you should subtype the "wood" stairs, maybe get a sprite

Erikafox commented on _maps/RandomRuins/BeachRuins/beach_crashed_starwalker.dmm:
```diff
@@ -0,0 +1,5515 @@
+"Xa" = (
+/obj/effect/decal/cleanable/dirt/dust,
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/dirt,
```
doubled up dirt

spockye replied:
does that cause issues?

Erikafox replied:
No

Erikafox reviewed (commented):
map review later

spockye committed:
misc fixes

makes a new wood stair subtype

switched out sci corpse and lathe for sec

fixed baby carp desc

fixed gps

Erikafox reviewed (changes requested):
![image](https://user-images.githubusercontent.com/94164348/213480350-f3bddeb6-fac7-4c31-8c21-a13a650d880b.png)
scatter some broken cables around here

Varedit ALL the APCs to have them start empty.
![StrongDMM_ezxNLnaIQN](https://user-images.githubusercontent.com/94164348/213480592-d1da0e64-ecc0-4719-af01-0933862a0159.png)
I'd give the chair a funny name so people know it's important.

spockye committed:
misc changes

adds cable scrap around the ripped off engine.

changes chair name, and description

spockye committed:
Merge branch 'master' into brach-ruin-2

spockye committed:
Merge branch 'master' into brach-ruin-2

Erikafox commented on code/datums/ruins/beachplanet.dm:
```diff
@@ -28,3 +28,9 @@
+	name = "Crashed Starwalker"
+	id = "crashed_star"
+	description = "A Crashed Starwalker Class Pirate Ship. It's Crew is Long-Dead."
+	suffix = "beach_crashed_starwalker"
```
beach_crashed_starwalker.dmm

Erikafox commented:
Please do not mark the "I tested this" checkmark if you didn't attempt to spawn it in. That means you didn't test it.

spockye committed:
fixes .dmm in  .dm

spockye committed:
Merge branch 'master' into brach-ruin-2

spockye committed:
Merge branch 'master' into brach-ruin-2

spockye committed:
Merge branch 'master' into brach-ruin-2

spockye committed:
Merge branch 'master' into brach-ruin-2

spockye committed:
Merge branch 'master' into brach-ruin-2

Erikafox commented:
![image](https://user-images.githubusercontent.com/94164348/216677322-6d70e96d-458f-4dfb-9013-1539d7a30aa1.png)
This uses the cursed sand.

spockye committed:
Update beach_crashed_starwalker.dmm

spockye committed:
sand fixes

Erikafox committed:
Merge branch 'master' into brach-ruin-2

Signed-off-by: Bjarl <94164348+Bjarl@users.noreply.github.com>

Erikafox committed:
Merge branch 'master' into brach-ruin-2

Signed-off-by: Bjarl <94164348+Bjarl@users.noreply.github.com>

Erikafox committed:
Merge branch 'master' into brach-ruin-2

Signed-off-by: Bjarl <94164348+Bjarl@users.noreply.github.com>

Erikafox committed:
Merge branch 'master' into brach-ruin-2
